### PR TITLE
Localization of views - about page 

### PR DESCRIPTION
**Pull request by pavanjoshi914**
localization of views ---.> about page

### Describe the changes you have made in this PR -

* ERB template - app/views/about/index.html.erb
* yml directory - config/locales/views/about/

### Screenshots of the changes (If any) -

image

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: config/locales/views/about/en.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    main_heading: "ABOUT"
+    main_description: "Learn about the awesome people behind CircuitVerse"
+    product_details: "CircuitVerse is a product developed by students at IIIT-Bangalore.
+          It aims to provide a platform where circuits can be designed and simulated using a graphical user interface.
+          While users can design complete CPU implementations within the simulator, the software is designed primarily for educational use.
+          CircuitVerse is an open-source project with an active community. Checkout the contribute page for more detail."
+    terms_of_service: "Terms of Service"
+    privacy_policy: "Privacy Policy"
+    email_us_heading: "Email us at"
+    email_us_description: "support@circuitverse.org"
```
Comment (tachyons) [inline config/locales/views/about/en.yml:13]: Email will be same in all languages, no need to translate

Comment (pavanjoshi914) [reply config/locales/views/about/en.yml]: changes done accordingly

**Comment by koolgax99** [comment]
We can write चित्रितोप योजक साधन  for the graphical user interface.
For the last sentence we could use अधिक जानकारी के लिए योगदान पृष्ठ की जांच करें |


can i make the pr or this @pavanjoshi914?

**Comment by pavanjoshi914** [comment]
> We can write चित्रितोप योजक साधन for the graphical user interface.
> For the last sentence we could use अधिक जानकारी के लिए योगदान पृष्ठ की जांच करें |
> 
> can i make the pr for this @pavanjoshi914?

we have to keep some english based technicals as it is in hindi as no one understand ```चित्रितोप योजक साधन ```

for the last sentence it is a button so keeping text short and concise is more important in perspective of UI.

I think current one are fine for both.

@tachyons

**Comment by tachyons** [comment]
Readability is more important than being pure Hindi words, I don't have expertise to comment on Hindi, so tagging @shreyaprasad

**Comment by ShreyaPrasad1209** [comment]
@pavanjoshi914 There are few things I would like to suggest:

1. Change last line to "अधिक जानकारी के लिए योगदान पृष्ठ dekhiye (in hindi)". this is more short and concise.
2. Instead of डिज़ाइन use बनावट, रचना. 
3.  Change project to परियोजना.
4. In some placed like chatro, prathmic - you have used some different kind of syntax. I will write it on paper and upload the syntax we normally use in Hindi.

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@pavanjoshi914 There are few things I would like to suggest:

1. Change last line to "अधिक जानकारी के लिए योगदान पृष्ठ dekhiye (in hindi)". this is more short and concise.
2. Instead of डिज़ाइन use बनावट, रचना. 
3.  Change project to परियोजना.
4. In some placed like chatro, prathmic - you have used some different kind of syntax. I will write it on paper and upload the syntax we normally use in Hindi.

**Comment by pavanjoshi914** [comment]
> @pavanjoshi914 There are few things I would like to suggest:
> 
> 1. Change last line to "अधिक जानकारी के लिए योगदान पृष्ठ dekhiye (in hindi)". this is more short and concise.
> 2. Instead of डिज़ाइन use बनावट, रचना.
> 3. Change project to परियोजना.
> 4. In some placed like chatro, prathmic - you have used some different kind of syntax. I will write it on paper and upload the syntax we normally use in Hindi.

words like ```पृष्ठ```   ``` परियोजना```  seems not been used when we talk in hindi in our day to day life. 

like we say page as page "page (hindi)" and not prushth. 

I am saying this in perspective of keeping translations as readable to wider audience  and as simple as possilble.

that is why is used alternate names for them , like project kept as project only 
 
 alternate words for the project - 
image

**Comment by koolgax99** [comment]
> > @pavanjoshi914 There are few things I would like to suggest:
> > 
> > 1. Change last line to "अधिक जानकारी के लिए योगदान पृष्ठ dekhiye (in hindi)". this is more short and concise.
> > 2. Instead of डिज़ाइन use बनावट, रचना.
> > 3. Change project to परियोजना.
> > 4. In some placed like chatro, prathmic - you have used some different kind of syntax. I will write it on paper and upload the syntax we normally use in Hindi.
> 
> words like `पृष्ठ` ` परियोजना` seems not been used when we talk in hindi in our day to day life.
> 
> like we say page as page "page (hindi)" and not prushth.
> 
> I am saying this in perspective of keeping translations as readable to wider audience and as simple as possilble.
> 
> that is why is used alternate names for them , like project kept as project only
> 
> alternate words for the project -
> image

i guess yojna would be a  better word here

**Comment by pavanjoshi914** [comment]
@ShreyaPrasad1209 changes for points 1, 2, 3 done accordingly please have a check!

regarding point no 4. their is some unknown glitch , when i paste translations anywhere they seems correct but in yml they are not interpreted correctly so we can ingnore that for now.

File: config/locales/views/about/hi.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+hi:
+  about:
+    title: "सर्किटवर्स - हमारे बारे में"
+    main_heading: "सर्किटवर्स के बारे में जाने"
+    main_description: "सर्किटवर्स के पीछे योगदानकर्ताओं के बारे में जानें"
+    product_details: "सर्किटवर्स आईआईआईटी-बैंगलोर के छात्रों द्वारा विकसित एक प्रोडक्ट है।
+          इसका उद्देश्य एक ऐसा मंच प्रदान करना है जहां ग्राफिकल यूजर इंटरफेस का उपयोग करके सर्किट की रचना और उसका सिमुलेशन किया जा सकता है।
+          जबकि उपयोगकर्ता सिम्युलेटर के द्वारा पूर्ण सीपीयू कार्यान्वयन की रचना कर सकते हैं, सॉफ्टवेयर की रचना प्राथमिक रूप से शैक्षिक उपयोग के लिए की गयी है।
```
Comment (ShreyaPrasad1209) [inline config/locales/views/about/hi.yml:8]: Here remove जबकि and add “lekin” before software.

**Review by ShreyaPrasad1209 — CHANGES_REQUESTED** [review]
@pavanjoshi914 great work! I have suggested one change, have a look and let me know your thoughts on it.

File: config/locales/views/about/hi.yml (hunk 2 above)
Comment (pavanjoshi914) [reply config/locales/views/about/hi.yml:8]: 'lekin' means 'but' , the sentence ```while we can design cpu in simulator``` which represents a particular features. 

in my opinion using ```but we can design cpu in simulator``` in hindi  doesnt give any context of representation of features

above explanation sounds good?

Comment (ShreyaPrasad1209) [reply config/locales/views/about/hi.yml:8]: Yeah 👍🏻

**Review by ShreyaPrasad1209 — APPROVED** [review]

**Comment by pavanjoshi914** [comment]
@tachyons  we are good to go here , please have a check on this

**Pull request merged**